app/controllers/provider_foreman_controller.rb

```diff
@@ -536,8 +559,14 @@ def provider_node(id, model)
         @right_cell_text = _("%{model} \"%{name}\"") % {:name  => provider.name,
                                                         :model => "#{ui_lookup(:tables => "configuration_profile")} under #{record_model}"}
       when "ManageIQ::Providers::AnsibleTower::ConfigurationManager"
-        @right_cell_text =
-          _("%{model} \"%{name}\"") % {:name => provider.name, :model => "#{record_model}"}
+        options = {:model => "ManageIQ::Providers::ConfigurationManager::InventoryGroup", :match_via_descendants => ConfiguredSystem}
+        options[:where_clause] = ["ems_id IN (?)", provider.id]
```

Comment:
Hmm... provider is an `Manager` / `ExtManagementSystem` here... again :confused: